--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1811,12 +1811,14 @@
       _py_revision = string_replace(_py_revision, "'", "")
       _py_subrevision =
           string_replace(_rust_revision_lines[1], "RUST_SUB_REVISION = ", "")
-      _expected_rust_revision = "$_py_revision-$_py_subrevision"
+      # DISABLED for custom toolchain extraction
+      # _expected_rust_revision = "$_py_revision-$_py_subrevision"

       # Ensure the rust version matches the clang version.
-      assert(
-          filter_include([ rustc_revision ],
-                         [ "*-$_expected_rust_revision-*" ]) != [],
-          "rustc_revision=\"$rustc_revision\" but update_rust.py expected \"$_expected_rust_revision\". Run \"gclient sync\"?")
+      # DISABLED for custom toolchain extraction
+      # assert(
+      #     filter_include([ rustc_revision ],
+      #                    [ "*-$_expected_rust_revision-*" ]) != [],
+      #     "rustc_revision=\"$rustc_revision\" but update_rust.py expected \"$_expected_rust_revision\". Run \"gclient sync\"?")
     }

     # This is here so that all files get recompiled after a clang roll and
